emails/templates/email-verification.js

```diff
@@ -35,15 +35,13 @@ module.exports = function() {
       Verify your email to receive email notifications about important news
       and upcoming tasks.
 
-      To complete the verification process, please click on the button below.
```

Comment:
Not sure if this change was meant to be made; AFAIK we still have that text in the email?

Reply:
Well there is no "button" in the text version of email so I removed it

Reply:
😅good catch!